### ISSUE TITLE
Mac VoiceOver focus is stuck on initial option in multiple selection

### ISSUE THREAD

**Issue by Zorrow2008** [angular/angular#42902]
### Which @angular/* package(s) are the source of the bug?

Don't known / other

### Is this a regression?

No

### Description

When MacOS VoiceOver is enabled and you open up at a multiple selection mat-select form, the black outline (VoiceOver focus) is always stuck to the initial option that opens up. You can tab and arrow key between options and regular focus works fine and VoiceOver even voices things correctly. However the black outline representing the VoiceOver focus is always stuck on the initial option that opens up when expanding the multiple select.

### Please provide a link to a minimal reproduction of the bug

https://material.angular.io/components/select/overview#multiple-selection

### Please provide the exception or error you saw

```true
Steps to reproduce:
1) Go to https://material.angular.io/components/select/overview#multiple-selection
2) Enable MacOs VoiceOver
3) Click on the multiple selection and tab through options
4) Notice how the black rectangle (voiceover focus) is always set to the initial option that opens up.
```

### Please provide the environment you discovered this bug in

```true
OS: macOS Big Sur Version 11.3.1
```
